background/redux-slices/ui.ts

```diff
@@ -392,6 +394,22 @@ export const setSelectedNetwork = createBackgroundAsyncThunk(
   },
 )
 
+export const toggleShowTestNetworks = createBackgroundAsyncThunk(
+  "ui/toggleShowTestNetworks",
+  async (value: boolean, { dispatch, getState }) => {
```

Comment:
```suggestion
  async (updatedValue: boolean, { dispatch, getState }) => {
```

Perhaps for clarity? Or even `updatedShowTestNetworksValue` or similar?